testing.go

```diff
@@ -288,27 +289,29 @@ func (c *cluster) WaitEventChan(filter FilterFn, timeout time.Duration) <-chan s
 			select {
 			case <-timeoutCh:
 				return
-
+			case <-doneCh:
```

Comment:
Hey, could you ELI5 the difference between the `timeoutCh` and `doneCh`  🙏

Reply:
`WaitEventChan` spawns a goroutine but there was no way to stop it. It only stopped before on timeout, but if we know that we don't need it anymore, it can now be stopped by closing doneCh. There are other ways of doing this, eg using context.

Reply:
Thank makes sense, tyvm <3

Reply:
Switched to context.